src/services/integration-elements.ts

```diff
@@ -961,6 +962,38 @@ export async function getAPINSWLiquorLicence(
   }
 }
 
+const CP_HCMS_TOKEN_KEY = 'CP_HCMS_TOKEN'
+function getCPHCMSTokenFromStorage() {
+  const localStorageValue = localStorage.getItem(CP_HCMS_TOKEN_KEY)
```

Comment:
Need to join the `formId` to this to allow different menu items with different content.